### DISCUSSION TITLE
"def" and "let" how implement it in use

### DISCUSSION THREAD

**Discussion by MohammedTayebi** [useocl/use#30] [General]

**Comment by h-man2** [comment]
Sorry that I didn't answer until now. I forgot to activate notifications for discussion... :-(

The OCL let-Expression is supported by USE:
```
let i:Integer = 5 in i * i
```

The keyword `def` is not supported, but you can define query operations using the syntax shown in this [example](https://github.com/useocl/use/blob/master/manual/main.md#syntax-3):
```
class Lemon
operations
  squeeze(i : Integer) : Integer = i + 1
end
```

Here, `i + 1` is the (query) body of the operation.

**Reply by MohammedTayebi** [reply]
Thanks it is useful
thanks very much
